### DISCUSSION TITLE
Does Presidio support for multi line text?

### DISCUSSION THREAD

**Discussion by chamiw2008** [microsoft/presidio#799] [Q&A] [answered]
Tried Presidio on latest Dockers.

Send the following text;
String inputText = "Mr. Cha Wee a 46year old male patient from Southampton presented with severe left loin colic pain 3 days duration.\n"
                + "Sent home on painkillers and plan for X-ray KUB and USS Abdomen ASAP. \n"
                + "He can be contacted on cha2008@gmail.com\n"
                + "or over the phone on +94773003053 and local number 077512 962 769";

Get an error 500!

Overall setup works fine with single liners such as ;
String inputText = "John Smith drivers license is AC432223";

Any thoughts how to allow multi line support ?

Thanks,

**Comment by omri374** [comment, marked as answer]
Thanks @chamiw2008, we'll look into this. Created an issue: https://github.com/microsoft/presidio/issues/800
